bundle/src/main/java/com/adobe/acs/commons/httpcache/engine/package-info.java

```diff
@@ -18,6 +18,6 @@
  * #L%
  */
 
-@org.osgi.annotation.versioning.Version("3.3.0")
+@org.osgi.annotation.versioning.Version("4.0.0")
```

Comment:
Is it entirely necessary to have such a large API version increase?  @justinedelson, thoughts on this?  I'm not adverse but I would rather refrain if possible.  That would mean that either this PR would have to wait for the next minor version bump potentially, or we would have to find a way to adapt these changes so that such a large increase isn't necessary.

Reply:
This plugin forced me to bump it up with this high number :( I am not sure of the rules or how this thing works exactly.. some insight would be good indeed